org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WTableDirEditor.java

```diff
@@ -259,7 +259,7 @@ private void init()
     {
     	ZKUpdateUtil.setWidth(getComponent(), "200px"); 
         getComponent().setAutocomplete(true);
-        getComponent().setAutodrop(false);
+        getComponent().setAutodrop(true);
```

Comment:
this change is not related to this ticket, please remove it.

Reply:
ok, done